storage/service_account/client_endpoint/set_client_endpoint.go

```diff
@@ -25,19 +25,47 @@ import (
 )
 
 // setClientEndpoint sets the request endpoint.
-func setClientEndpoint(w io.Writer, customEndpoint string) error {
-	// customEndpoint := "https://my-custom-endpoint.example.com/storage/v1"
+func setClientEndpoint(w io.Writer, customEndpoint string, opts ...option.ClientOption) error {
+	// customEndpoint := "https://my-custom-endpoint.example.com/storage/v1/"
+	// opts := []option.ClientOption
```

Comment:
You can test this out locally to make sure the type is correct but I think it would have to be `[]option.ClientOption{}`

Reply:
yes missed that, thanks!